### ISSUE TITLE
Allow to filter flats intended at the elderly ("Senioren")

### ISSUE THREAD

**Issue by neopostmodern** [neopostmodern/wohnungsbot#29]
Data sample:

```txt
Seniorenwohnung ab 55 Jahren!

Wohnen für Senioren Ü55

Seniorenwohnung ab 55 Jahren! Mit Dusche!

Sanierte Seniorenwohnung ab 55 Jahre

Sanierte Seniorenwohnung ab 60 Jahre

Sanierte Wohnung für Senioren ab 55 Jahre
```

**Comment by valoricDe** [comment]
This one is a bit tricky. As the bot is only looking on the first page. But it might be that a flat with title `Senioren` is not on the first page. Ideally we could search for a keyword. But I guess that's not possible

**Comment by neopostmodern** [comment]
Closed with #42
